index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel=" stylesheet" href="style/style.css">
+    <title>Portifolio</title>
+</head>
+
+<body>
+    <div class="container">
+        <header>            
+            <div class="toolbar">
+                <div class="logo">
+                    <h2>My Logo</h2>
+                </div>
+                <div class="menu"><img src="assets/Union.png" alt="Menu"></div>
+            </div>
```

Comment:
- [ ] The hamburger button must have a `nav` tag

Reply:
:+1: Issue addressed successfully :heavy_check_mark:  

- [ ] [Optional] You can just use [fontawesome]() icons, instead of a bare image, when you use icons, you page size is optimized hence it increases the performance